neqo-http3/tests/httpconn.rs

```diff
@@ -379,3 +412,45 @@ fn zerortt() {
     exchange_packets(&mut hconn_c, &mut hconn_s, out.dgram());
     process_client_events(&mut hconn_c);
 }
+
+#[test]
+fn test_fetch_noresponse_will_idletimeout() {
```

Comment:
```suggestion
/// When a client has an outstanding fetch, it will send keepalives.
/// Test that it will successfully run until the connection times out.
fn fetch_noresponse_will_idletimeout() {
```